project/Dependencies.scala

```diff
@@ -22,6 +22,8 @@ object Dependencies {
   val lmdb                 = "org.lmdbjava"           % "lmdbjava"         % "0.6.0"
   val shapeless            = "com.chuusai"            %% "shapeless"       % "2.3.2"
   val scalaCheck           = "org.scalacheck"         %% "scalacheck"      % "1.13.4" % "test"
+  val guav                 = "com.google.guava"       % "guava"            % "16.0"
```

Comment:
Not sure I understood this change?

Reply:
It was a fix for this error:
https://travis-ci.org/rchain/rchain/jobs/343519746#L979